appinventor/blocklyeditor/src/generators/yail/math.js

```diff
@@ -91,6 +91,14 @@ Blockly.Yail['math_multiply'] = function() {
   return Blockly.Yail.math_arithmetic_list("MULTIPLY",this);
 };
 
+Blockly.Yail['math_bitwise_and'] = function() {
+  return Blockly.Yail.math_arithmetic("BITAND",this);
```

Comment:
I'm going to be picky here and request the space between the `,` and `this`. I realize the previous code does it but it's actually against the style guide.